src/taco-cli/cli/taco.ts

```diff
@@ -73,16 +73,13 @@ class Taco {
             new CheckForNewerVersion().showOnExitAndIgnoreFailures();
 
             var parsedArgs: IParsedArgs = Taco.parseArgs(process.argv.slice(2));
-            var commandProperties: ICommandTelemetryProperties = {};
 
             Taco.runWithParsedArgs(parsedArgs)
                 .then(function (telemetryProperties: ICommandTelemetryProperties): void {
                     if (parsedArgs.command) {
-                        commandProperties = telemetryProperties;
+                        // Send command success telemetry
+                        telemetryHelper.sendCommandSuccessTelemetry(parsedArgs.commandName, telemetryProperties, parsedArgs.args);
```

Comment:
We already handle that case (routed commands to Cordova) in runWithParsedArgs. So, this change looks good.